src/OmniSharp/Api/Intellisense/OmnisharpController.Intellisense.cs

```diff
@@ -130,15 +136,12 @@ private AutoCompleteResponse MakeAutoCompleteResponse(AutoCompleteRequest reques
                 var snippetGenerator = new SnippetGenerator();
                 snippetGenerator.IncludeMarkers = true;
                 snippetGenerator.IncludeOptionalParameters = includeOptionalParams;
-                response.Snippet = snippetGenerator.GenerateSnippet(symbol);
+                response.Snippet = snippetGenerator.Generate(symbol);
             }
 
             if (request.WantMethodHeader)
             {
-                var snippetGenerator = new SnippetGenerator();
-                snippetGenerator.IncludeMarkers = false;
-                snippetGenerator.IncludeOptionalParameters = includeOptionalParams;
-                response.MethodHeader = snippetGenerator.GenerateSnippet(symbol);
+                response.MethodHeader = symbol.ToDisplayString(SymbolDisplayFormat.MinimallyQualifiedFormat);
```

Comment:
you deleted 4 lines that were using SnippetGenerator previously, and now it is doing symbol.ToDisplayString (which is completely different ;) )